test/avocado/testlib_avocado/machineslib.py

```diff
@@ -236,11 +236,8 @@ def create_vm_by_ui(self,
             if element:
                 self.click(element)
 
-        if os_vender != 'unspecified':
-            self.select_by_text(self.wait_css('#vendor-select'))
-
-        if os_vender != 'unspecified' and os != 'other':
-            self.select_by_text(self.wait_css('#system-select'), os)
+        if os != 'other':
```

Comment:
Same question -- I thought "other" was a cockpit-ism that got removed in the previous commit; is it an actual valid value in libosinfo/libvirt?

Reply:
In this selenium test, they assume that the default os is other, see https://github.com/cockpit-project/cockpit/blob/master/test/avocado/testlib_avocado/machineslib.py#L205
In this case they don't set anything in the dialog which in our case works fine.
I think it will be cleaner to change the other to None in the selenium test.

Reply:
> > On the other hand you are able to filter results by typing some substring of the desired distro.
> 
> Right, I personally like this a lot, but I'm a very keyboard-affine person. I was just wondering what that means for mouse users.
> 
> Also, right now I can type arbitrary strings into that box, like "foo", and there is no error message. It should only be allowed to select an existing entry.

Oh good catch, it gives and 'No matches found' but then the dialog gets the value even if it's nonsense. Good catch.

Reply:
So this seems to be the default behavior of the component, see https://tinyurl.com/y22ckuym
And it's not overwritable,
I thnk I will be able to workaround it in a hacky way, since it's really weird to be able to submit the dialog with false data.

Reply:
Done